opm/input/eclipse/EclipseState/Aquifer/NumericalAquifer/SingleNumericalAquifer.cpp

```diff
@@ -95,6 +116,8 @@ namespace Opm {
     std::vector<NNCdata>
     SingleNumericalAquifer::aquiferConnectionNNCs(const EclipseGrid& grid, const FieldPropsManager& fp) const {
        std::vector<NNCdata> nncs;
+        if (this->cells_.size() == 0)
+            return nncs;
```

Comment:
I think we can use this check also for the function above

`    std::vector<NNCdata> SingleNumericalAquifer::aquiferCellNNCs() const` 

Then we do not need to cast `std::size_t` to `int`. 

Since you are there, `this->cells_.empty()` can be better readability.